### DISCUSSION TITLE
request rate

### DISCUSSION THREAD

**Discussion by RuSS512** [python-telegram-bot/python-telegram-bot#3865] [Q&A]
As far as I know, the bot itself sends a request to receive changes every second.

How to make the bot send such a request once every 24 hours, and not every second?

**Comment by Bibo-Joshi** [comment]
Hi. Neither of those two options is how it works. Please have a look at the parameter "timeout" of Bot.get_updates/Application.run_polling and search for the term "long polling" on your favorite search engine.
